dataframely/columns/datetime.py

```diff
@@ -317,28 +329,51 @@ def __init__(
             metadata=metadata,
         )
         self.resolution = resolution
+        self.time_zone = time_zone
 
     @property
     def dtype(self) -> pl.DataType:
-        return pl.Datetime()
+        return pl.Datetime(time_unit="us", time_zone=self.time_zone)
```

Comment:
As long as `time_unit` is not a parameter of `dy.Datetime`, let's drop the constant and take polars' default
```suggestion
        return pl.Datetime(time_zone=self.time_zone)
```